sfepy/examples/multi_physics/piezo_elasticity_macro.py

```diff
@@ -38,24 +27,25 @@ def post_process(out, pb, state, extend=False):
 
     state_dict = state.get_state_parts()
     displ = state_dict['u']
-    strain_qp = pb.evaluate('ev_cauchy_strain.i2.Omega(u)', mode='qp')
 
-    nodal_data = {
-        'u': displ.reshape((displ.shape[0] // dim, dim)),  # displacement
-        'strain': linear_projection(pb, strain_qp),  # strain
-    }
-    const_data = {
-        'phi': pb.conf.phi,  # el. potentials
+    displ = displ.reshape((displ.shape[0] // dim, dim))
+    macro = {
+        'u': ('val', 'svar', displ),
+        'strain': ('cauchy_strain', 'svar', displ),
+        '_phi': pb.conf.phi,
     }
+
     def_args = {
         'eps0': pb.conf.eps0,
         'filename_mesh': pb.conf.filename_mesh_micro,
     }
+
     pvar = pb.create_variables(['svar'])
 
-    recover_micro_hook_eps(pb.conf.filename_micro, rreg,
-                           pvar['svar'], nodal_data, const_data, pb.conf.eps0,
-                           define_args=def_args)
+    recover_micro_hook(pb.conf.filename_micro, rreg, macro, pb.conf.eps0,
+                       region_mode='el_centers', eval_mode='constant',
+                    #    region_mode='tiled', eval_mode='constant',
```

Comment:
Remove or comment what it does?

Reply:
`region_mode` and `eval_mode` are now the arguments of `define()`.